test/lab/process/code/CalcuatorTest.java

```diff
@@ -95,4 +95,16 @@ void testDivideZero() {
 
 	}
 
+	@Test
+	void testPower() {
+		int value1 = 5;
+		int value2 = 3;
+		double answer;
+
+		answer = calculator.power(value1, value2);
+
+		assertEquals(answer, 125);

```

Comment:
Best to source the Test case input (test data) and output (expected result) values from a Property file instead of hard coding